src/index.html

```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      data-qa="big"
+      class="form"
+    >
+      <img
+        src="./images/Search.svg"
+        alt="Search"
+        class="icon"
+      >
+      <input
+        type="search"
+        data-qa="keypress"
+        class="input"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
+    <form
```

Comment:
```suggestion
    </form>
    
    <form
```
that would be perfect 🤩